src/html/components/modal-box.html

```diff
@@ -0,0 +1,20 @@
+<div class="modal-overlay">
+	<div class="modal-content">
+		<button class="modal-close-button">
+			<span class="modal-close-button__row"></span>
+			<span class="modal-close-button__row"></span>
+		</button>
+		{% include "html/components/registration-block.html" %}
+	</div>
+	<div class="modal-gratitude gratitude">
```

Comment:
why do you need to different classes `modal-gratitude` and `gratitude` ?

Reply:
Done.